CMIP6_source_id.json

```diff
@@ -789,44 +802,50 @@
             "label_extended":"CNRM-CM6-1-HR",
             "model_component":{
                 "aerosol":{
-                    "description":"climatological fields computed by TACTIC (v2.0)",
-                    "nominal_resolution":""
+                    "description":"prescribed monthly fields computed by TACTIC_v2 scheme",
+                    "nominal_resolution":"100 km"
                 },
                 "atmos":{
-                    "description":"Arpege 6.2 (Tl359; 91 levels; top level 78.4 km)",
-                    "nominal_resolution":""
+                    "description":"Arpege 6.3 (Tl359, 181724 x 1; XXX x XXX longitude/latitude; 91 levels; top level 78.4 km)",
```

Comment:
The entry 181724 x 1; XXX x XXX longitude/latitude; needs to be updated to a single correct value